.github/workflows/build-and-deploy.yml

```diff
@@ -6,10 +6,13 @@ on:
   pull_request:
     branches:
       - master
+  schedule:
+    # Every hour on the hour.
+    - cron:  '*/60 * * * *'
```

Comment:
Why do we need to do this? Won't this unnecessarily generate cache churn for us when we re-build the site and flip origins every hour?

Reply:
Only if there are changes to be made. If the job runs and finds that pulumi/hugo@master hasn't changed, it'll exit silently without generating a PR.